certified-connectors/VirusTotal/apiDefinition.swagger.json

```diff
@@ -0,0 +1,3325 @@
+        ],
+        "summary": "Analysis an URL",
+        "description": "Retrieve information about a file or URL analysis",
+        "operationId": "VirusTotalAnalysisurlV3",
+        "parameters": [
+          {
+            "in": "formData",
+            "name": "url",
+            "description": "URL to be analyzed",
+            "required": true,
+            "type": "string",
+            "x-ms-summary": "file"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Ok",
+            "schema": {
+              "$ref": "#/definitions/urlReport"
+            }
+          }
+        }
+      }
+    },
+    "/api/v3/analyses/{id}": {
+      "get": {
+        "summary": "Retrieve information about a file or URL analysis",
+        "description": "Retrieve information about a file or URL analysis",
+        "operationId": "VirusTotalRetrieveInfo",
+        "parameters": [
+          {
+            "in": "path",
+            "name": "id",
+            "description": "Analysis identifier",
+            "required": true,
+            "type": "string",
+            "x-ms-summary": "id",
+            "x-ms-url-encoding": "single"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Ok",
+            "schema": {
+              "$ref": "#/definitions/analysesResponses"
+            }
+          }
+        }
+      }
+    },
+    "/api/v3/files/{id}": {
+      "get": {
+        "summary": "Retrieve information about a file",
+        "description": "Retrieve information about a file",
+        "operationId": "VirusTotalRetrieveInfoaboutFile",
+        "parameters": [
+          {
+            "in": "path",
+            "name": "id",
+            "description": "SHA-256, SHA-1 or MD5 identifying the file",
+            "required": true,
+            "type": "string",
+            "x-ms-summary": "id",
+            "x-ms-url-encoding": "single"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Ok",
+            "schema": {
+              "$ref": "#/definitions/fileReport"
+            }
+          }
+        }
+      }
+    },
+    "/api/v3/files": {
+      "post": {
+        "consumes": [
+          "multipart/form-data"
+        ],
+        "summary": "Upload and analyse a file",
+        "description": "Upload and analyse a file",
+        "operationId": "VirusTotalAnalyesFile",
+        "parameters": [
+          {
+            "in": "formData",
+            "name": "file",
+            "description": "File to be scanned",
+            "required": true,
+            "x-ms-summary": "filename",
+            "type": "file"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Ok",
+            "schema": {
+              "$ref": "#/definitions/filesReport"
+            }
+          }
+        }
+      }
+    }
+  },
+  "definitions": {
+    "analysesResponses": {
```

Comment:
This is huge... Do you need all properties or can it be resricted to few important/most used properties? If yes, how are you expecting uses to use it? This looks like a usability issue

Reply:
Yes, we have review the requirement with the PM, and discuss the large amount of properties, After double verify it have come to understanding that it must have all the properties

Reply:
Can you please connect your PM with me. I am concerned about the user experience